sphinx/application.py

```diff
@@ -1133,18 +1133,20 @@ def is_parallel_allowed(self, typ):
         """
         if typ == 'read':
             attrname = 'parallel_read_safe'
+            gerund_typ = __("reading")
         elif typ == 'write':
             attrname = 'parallel_write_safe'
+            gerund_typ = __("writing")
         else:
             raise ValueError('parallel type %s is not supported' % typ)
 
         for ext in itervalues(self.extensions):
             allowed = getattr(ext, attrname, None)
             if allowed is None:
                 logger.warning(__("the %s extension does not declare if it is safe "
-                                  "for parallel %sing, assuming it isn't - please "
+                                  "for parallel %s, assuming it isn't - please "
                                   "ask the extension author to check and make it "
-                                  "explicit"), ext.name, typ)
+                                  "explicit"), ext.name, gerund_typ)
```

Comment:
Is it no problem to replace a single word in sentence for translation?
I guess the following style is better for translation. what do you think?
```
if typ == 'read':
    message = __("the %s extension..."
                            " for parallel reading, ...")
elif typ == 'write':
    message = __("the %s extension..."
                            " for parallel writing, ...")
else:
    ...
```